addons/mail/__manifest__.py

```diff
@@ -217,6 +221,8 @@
             'mail/static/src/components/attachment_box/attachment_box.scss',
             'mail/static/src/components/attachment_list/attachment_list.scss',
             'mail/static/src/components/attachment_viewer/attachment_viewer.scss',
+            'mail/static/src/components/category/category.scss',
+            'mail/static/src/components/category_item/category_item.scss',
```

Comment:
try to keep component scss with js when possible (scss order is only important when making overrides or using variables/mixin that have to be defined first)